src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -298,7 +319,9 @@ def generate_help_text(self):
             command = new_command
 
         # get command/group help
-        if self.completer and command in self.completer.command_description:
+        if not command and self.recommender.enabled:
+            self.description_docs = u'Try [Space] or `next` to get Command Recommendation'
+        elif self.completer and command in self.completer.command_description:
```

Comment:
Add a comment to explain under what conditions we will display the promotion message of `az next` and under what conditions we will display the help description of commands